### DISCUSSION TITLE
Autoplay

### DISCUSSION THREAD

**Discussion by daleSOCI** [naver/egjs-flicking#544] [Q&A] [answered]
I've tried implementing autoplay on this slider in React, and I've followed the ultra basic source from the [Plugins docs](https://naver.github.io/egjs-flicking/Plugins). Here's the sandbox link: https://codesandbox.io/s/quirky-brown-hf593?file=/src/App.js

For some reason it plays once, then stops working. Am I missing something here?

**Comment by WoodNeck** [comment, marked as answer]
Hello @daleSOCI!
I've already made a PR that fixes this issue(#542), so it will be fixed in the next version.

As this issue happens only in the v4.2.4,
If you have to use it right away, please downgrade Flicking to 4.2.3 while waiting for it :)

**Reply by WoodNeck** [reply]
I've just released a new version that fixes this issue, please install v4.2.5 :)

**Reply by daleSOCI** [reply]
Awesome, thanks!
